### PR TITLE
Ignore null values on participant card header

### PR DESCRIPTION
**Pull request by sivaOSI**
@pemaOSI or @noraOSI Ready for review

### CODE REVIEW

File: app/assets/javascripts/components/screenings/ParticipantEditView.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {Link} from 'react-router'
 const ParticipantEditView = ({participant}) => (
   <div className='card edit double-gap-top' id={`participants-card-${participant.get('id')}`}>
     <div className='card-header'>
-      <span>{`${participant.get('first_name')} ${participant.get('last_name')}`}</span>
+      <span>{`${[participant.get('first_name'), participant.get('last_name')].filter(Boolean).join(' ')}`}</span>
```
Comment (noraOSI) [inline app/assets/javascripts/components/screenings/ParticipantEditView.jsx]: probably don't want to render an empty span when name is null

File: app/assets/javascripts/components/screenings/ParticipantShowView.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import {Link} from 'react-router'
 const ParticipantShowView = ({participant, onEdit}) => (
   <div className='card show double-gap-top' id={`participants-card-${participant.get('id')}`}>
     <div className='card-header'>
-      <span>{`${participant.get('first_name')} ${participant.get('last_name')}`}</span>
+      <span>{`${[participant.get('first_name'), participant.get('last_name')].filter(Boolean).join(' ')}`}</span>
       <Link aria-label='Delete participant' className='pull-right' href='#'>
```
Comment (noraOSI) [inline app/assets/javascripts/components/screenings/ParticipantShowView.jsx]: same as above. probably don't want to render an empty span when name is null

File: spec/javascripts/components/screenings/ParticipantEditViewSpec.jsx [hunk 3]
```diff
@@ -51,3 +51,22 @@ describe('ParticipantEditView', () => {
...
+      gender: 'female',
+      ssn: 'ssn-1',
+    })
+    component = shallow(<ParticipantEditView participant={participant} />)
+  })
+
+  it('renders the participant header corectly with null last name', () => {
+    expect(component.find('.card-header').text()).not.toContain('null')
+  })
+})
```
Comment (noraOSI) [inline spec/javascripts/components/screenings/ParticipantEditViewSpec.jsx]: Add test to make sure span isn't rendered when name is null.

File: app/assets/javascripts/components/screenings/ParticipantShowView.jsx [hunk 4]
```diff
@@ -4,10 +4,14 @@ import React from 'react'
...
 import {Link} from 'react-router'
 
-const ParticipantShowView = ({participant, onEdit}) => (
+const ParticipantShowView = ({participant, onEdit}) => {
+  const name = [participant.get('first_name'), participant.get('last_name')].filter(Boolean).join(' ')
+  return (
   <div className='card show double-gap-top' id={`participants-card-${participant.get('id')}`}>
     <div className='card-header'>
-      <span>{`${participant.get('first_name')} ${participant.get('last_name')}`}</span>
+      { name &&
```
Comment (pemaOSI) [inline app/assets/javascripts/components/screenings/ParticipantShowView.jsx]: Why can't this be all in one line?

Comment (sivaOSI) [inline app/assets/javascripts/components/screenings/ParticipantShowView.jsx]: we fixed it.
